lib/common/client/configuration/base.rb

```diff
@@ -108,13 +108,22 @@ def breakers_service
             end
           end
 
-          @service = Breakers::Service.new(
+          @service = create_new_breakers_service(matcher, exception_handler)
+        end
+
+        def create_new_breakers_service(matcher, exception_handler)
+          Breakers::Service.new(
             name: service_name,
             request_matcher: matcher,
+            error_threshold: breakers_error_threshold,
             exception_handler: exception_handler
           )
         end
 
+        def breakers_error_threshold
```

Comment:
Can we add a comment making clear that this is `The percentage of errors over which an outage will be reported.`, just to spare someone the need to read breakers documentation?

Reply:
done